superset-frontend/src/profile/components/App.tsx

```diff
@@ -41,62 +45,46 @@ export default function App({ user }: AppProps) {
         </Col>
         <Col xs={24} md={18}>
           <Tabs centered>
-            <Tabs.TabPane
+            <StyledTabPane
               key="1"
```

Comment:
I know this isn't part of this PR specifically and it's really just a nit but since we are here it would be cool to use non-numeric keys. We could use humanly readable keys with the names of the tabs.